squad_client/core/models.py

```diff
@@ -40,9 +40,13 @@ def __fill__(self, klass, results):
         objects = {}
         for result in results:
             obj = klass()
-            attrs = klass.attrs if len(klass.attrs) else [attr for attr in result.keys()]
+            attrs = klass.attrs if len(klass.attrs) else [attr.replace(' ', '_') for attr in result.keys()]
```

Comment:
you're re-adding the bug I resolved. Any reason for that?

Reply:
@mwasilew my bad :/  
This came back in from head reset i my branch while i was splitting the PR in two.